tests/testthat/test-reporter.R

```diff
@@ -1,3 +1,9 @@
+
+# Disable srcrefs because they differ across systems
+withr::local_options(list(rlang_trace_format_srcrefs = FALSE))
```

Comment:
Could you take a look at trimming the srcref path when it's a child of the working directory? (i.e. at least file an issue in rlang)

Reply:
Sure, but in any case we need to disable the srcrefs in unit tests, otherwise we'll get failures when the pointed code is updated, and that can be in other packages like rlang.

Reply:
Also Travis will typically not have srcrefs for packages like rlang, while they may be present locally.

Reply:
We now format srcrefs relative to `TESTTHAT_DIR`, see https://github.com/r-lib/testthat/pull/921/commits/7eb3640a654ebaf0871135668b8b5fee4cd139b4.

Reply:
I think we should probably figure out a way to enable source references in at least a limited set of tests, maybe constructing them by hand or similar, even if most of the tests need it to be disabled due to the above issues.